lib/logger/lib/logger/translator.ex

```diff
@@ -324,12 +324,19 @@ defmodule Logger.Translator do
         ["\nPostponed: #{inspect(postponed, inspect_opts)}"]
 
     if min_level == :debug do
+      state_info =
+        case state do
+          {state, data} ->
```

Comment:
Couldn't this be a user defined tuple then? That we think is State+Data, but it is just something else? How does Erlang/OTP report those? Is there another key we could look at instead?

Reply:
Good call, I should really always ask myself "what would Erlang do?" 😄

Indeed, it seems there is no such thing as State-Data unpacking, just `State`: https://github.com/erlang/otp/blob/OTP-27.0/lib/stdlib/src/gen_statem.erl#L4554-L4616. Will fix accordingly.

Reply:
[dadb797](https://github.com/elixir-lang/elixir/pull/13684/commits/dadb7974bafa99cd28e781a71544c957edaf9893) ✅